BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/DiagnosticsModule/DiagnosticsEventListener.cs

```diff
@@ -81,5 +80,39 @@ protected override void OnEventSourceCreated(EventSource eventSource)
 
             base.OnEventSourceCreated(eventSource);
         }
+
+        /// <summary>
+        /// This method checks if the given EventSource Name matches known EventSources that we want to subscribe to.
+        /// </summary>
+        private static bool ShouldSubscribe(EventSource eventSource)
```

Comment:
What exactly is the net effect of this change?

Reply:
This is collecting telemetry not related to our SDK. I'll send you the information